### DISCUSSION TITLE
Workflows For PocketBase

### DISCUSSION THREAD

**Discussion by bytekai** [pocketbase/pocketbase#4313] [Show and tell]
https://tryflot.com

Hey, I've previously talked about making a workflow engine for PocketBase, I'm happy to say that the core is done and people are using it already!

![image](https://github.com/pocketbase/pocketbase/assets/73543613/ab30b270-4e73-4589-98eb-5759248cf594)

Be warned its still quite early so you might face some bugs!

I have many plans of adding diffrent kind of utility nodes and integrations that would make developing applications much smoother and faster!

If any body wants to try it out join our discord.
https://discord.gg/nvDtHjdKsd

**Comment by starredev** [comment]
Tbh this is a nice feature to have in pocketbase aswell
